stats/cloud/collector.go

```diff
@@ -441,6 +456,19 @@ func (c *Collector) flushHTTPTrails() {
 	c.aggrBuckets = map[int64]aggregationBucket{}
 	c.bufferSamples = append(c.bufferSamples, newSamples...)
 }
+
+func (c *Collector) shouldStopSendingMetrics(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	if errResp, ok := err.(ErrorResponse); ok && errResp.Response != nil {
+		return errResp.Response.StatusCode == http.StatusForbidden && errResp.Code == 4
```

Comment:
I'm still wondering about the meaning of this magic `4` here. Can we make it a const in `stats/cloud/errors.go`? The name should at least document its meaning.

Reply:
@imiric it is the code from backend API, I think so.